src/net/sourceforge/kolmafia/AdventureResult.java

```diff
@@ -437,6 +437,9 @@ public String getName() {
       case ItemPool.PUNCHCARD_SPHERE:
         return AdventureResult.punchCardName(this.id);
 
+      case ItemPool.UNBREAKABLE_UMBRELLA:
+        return this.name + "(" + Preferences.getString("umbrellaState") + ")";
```

Comment:
Presumably you want " (" rather than "("

Reply:
Incredible catch.